src/Mews.Fiscalizations.All.props

```
@@ -3,6 +3,6 @@
     <AddSyntheticProjectReferencesForSolutionDependencies>false</AddSyntheticProjectReferencesForSolutionDependencies>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="FuncSharp" Version="[6.0.0, 7)" />
+    <PackageReference Include="FuncSharp" Version="7.0.1" />
```

Shouldn't it be this?
```suggestion
    <PackageReference Include="FuncSharp" Version="[7.0.0-preview.1]" />
```

---

It should. I was reverting the packages, so I forgot about this. 
I did use the correct one in Globalization though 😄

---

That's where I copied it from 😅